.vscode/launch.json

```diff
@@ -65,6 +65,14 @@
             "name": "Debug prepare-che-operator.js script",
             "program": "${workspaceFolder}/prepare-che-operator-templates.js"
           },
-
+        {
```

Comment:
It turns out that we already have `Attach to Remote` configuration in the `launch.json`
So, this changes are unnecessary, pls revert them.
Also, pls remove the following configuration from the `launch.json` as well
* `Debug prepare-che-operator.js script`
* `Debug Openshift Install`